x-pack/plugins/monitoring/public/legacy_shims.ts

```diff
@@ -37,10 +37,12 @@ export interface KFetchKibanaOptions {
   prependBasePath?: boolean;
 }
 
+const angularNoop = () => { throw new Error('Angular has been removed.') }
```

Comment:
I guess that we would need to remove this when we remove Angular. Could we add a comment like `TODO: remove when angular is removed` to not forget it?